---
bg_image: images/symposium/gault.png
category: ["eDNA", "Community ecology"]
description: by Alexis Carteron and Simon Morvan
draft: false
image: images/symposium/edna.png
information:
- info: English
  label: Language
- info: R and RStudio
  label: Material required
- info: R Symposium 2018
  label: Instructed
- info: Alexis Carteron and Simon Morvan
  label: Contributed by
title: eDNA metabarcoding
---

## eDNA metabarcoding: From raw data to RDA

With the democratisation of high-throughput sequencing, the use of DNA as an identification method has become standard practice. The marker genes (16S rRNA, 18S rRNA, ITS, etc.) can be compared to databases and give an overview of the communities present in your samples. No more cultivating, isolating and identifying based on morphology and chemical reactions ! However time-gaining DNA sequencing may be, the full process of turning raw DNA reads into exploitable taxonomic units can be tricky. Due to the quick turnover in bioinformatics techniques and the profusion of methods available, it can be hard to decide on a pipeline. In this workshop, we would like to present to you a complete reproducible workflow in R dedicated to the processing of raw DNA sequences using the DADA2 package, followed by community analysis and visualisation using the Phyloseq package.

### Workshop material

* The workshop material can be accessed [here](https://alexiscarter.github.io/metab/).